go.mod

```diff
@@ -1,20 +1,19 @@
 module github.com/scylladb/scylla-bench
 
-go 1.17
+go 1.23
```

Comment:
why are we updating the go version in this PR ?

if it's about packing of s-b, we should be touching the code or dependencies, we should be doing it on it's own PR, and get it tested

Reply:
Will remove it